cmd/minishift/cmd/hostfolder/sftpd.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
```

Comment:
2018

Reply:
well, it depends. I created this file last year.